cypress/integration/manualTests/zenodo.ts

```diff
@@ -0,0 +1,173 @@
+ *    limitations under the License.
+ */
+ import { goToTab } from '../../support/commands';
+
+function testID_Alpha() {
+  let text = 'zenodo-test-workflow';
+  const possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+
+  for (let i = 0; i < 10; i++) {
+    text += possible.charAt(Math.floor(Math.random() * possible.length));
+  }
+  return text;
+}
+
+// Locate a hosted workflow for which to create a DOI; if the specific hosted
+// workflow is not present then create it. Then snapshot the workflow and request
+// a DOI
+// You should link your Zenodo credentials before running the test
+describe('Create Zenodo DOI for workflow version', () => {
+    it('Should be able to create DOI', () => {
+
+      cy.visit('/login', {timeout: 30000});
+      cy.wait(2000);
+
+      cy.contains('Login with GitHub').click();
+      cy.wait(5000);
+
+      const organization = 'dockstore.org';
+      const zenodoWorkflow = 'zenodo-doi-test';
+      const wdlDescriptorType = 'WDL';
+
+      // Get the user name from the account drop down list so
+      // so we can construct a url to the hosted test workflow
+      // then try to go to the hosted test workflow
+      // cy.get('[data-cy=dropdown-main]').invoke('text').then((text => {
+      cy.get('[class=hidden-xs]').invoke('text').then((text => {
+        const userName = text.toString().trim().substr(0, text.toString().indexOf(' '));
+        cy.visit('/my-workflows/' + organization + '/' + userName + '/' + zenodoWorkflow);
+        // Wait a long time for the my workflows page to load
+        // Sometimes this takes longer than 30 seconds
+        cy.wait(60000);
+      }));
+
+      // Look at the workflow path at the top of the page and check to
+      // see that we got to the test hosted workflow. If user does not yet
+      // have the test workflow, the workflow path will not have the name
+      // of the test workflow in it. In that case create a hosted test workflow.
+      cy.get('[id=workflow-path]').invoke('text').then((workflowPath => {
+        let workflowName = workflowPath.toString();
+        workflowName = workflowPath.toString().substring(workflowPath.toString().lastIndexOf('/') + 1);
+        // accounts for no colon because an empty string is falsy, therefore it returns workflowPath
+        // https://stackoverflow.com/questions/4250364/how-to-trim-a-file-extension-from-a-string-in-javascript
+        workflowName = workflowName.substring(0, workflowName.lastIndexOf(':')) || workflowName;
+
+        if (workflowName !== zenodoWorkflow) {
+          // Create the Zenodo test hosted workflow
+          cy.get('[id=registerWorkflowButton]').click();
+          // Select the Use CWL, WDL or Nextflow from GitHub, Bitbucket, etc.' option to create a hosted workflow
+          cy.contains('Create and save CWL, WDL, or Nextflow on Dockstore.org').click();
+          // TODO: Fix this.  When 'Next' is clicked too fast, the next step is empty
```

Comment:
Does this comment still apply, or does your wait on the next line handle it?

Reply:
removed waits and fixed this